checks.d/process.py

```diff
@@ -11,6 +11,8 @@ class ProcessCheck(AgentCheck):
 
     SOURCE_TYPE_NAME = 'system'
 
+    CACHE_REFRESHING_FREQ = 20
```

Comment:
This is a convenient value for testing (quickly reached), but in a real situation we'll probably want to set it higher.